### PR TITLE
Numpy standardize

### PR DESCRIPTION
**Pull request by pvalle6**
- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``pre-commit``
locally, and understand that continuous integration checks will be used to
confirm the Biopython unit tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

Updated Celfile.py, bed.py, sam.py, __init__.py with #4238 reccomendations

### CODE REVIEW

File: Bio/Align/__init__.py [hunk 1]
```diff
@@ -2355,10 +2355,10 @@ def __repr__(self):
         distinguish different Alignment objects.  See help(id) for more
         information.
 
-        >>> import numpy
+        >>> import numpy as np
         >>> from Bio.Align import Alignment
         >>> alignment = Alignment(("ACCGT", "ACGT"),
-        ...                       coordinates = numpy.array([[0, 2, 3, 5],
+        ...                       coordinates = np.array([[0, 2, 3, 5],
```
Comment (peterjc) [inline Bio/Align/__init__.py:2361]: Can you also fix the following two lines to change their indentation to match?

**Review by peterjc — CHANGES_REQUESTED** [review]
One minor formatting change requested in the docstring example.

Unless you wanted to rebase this, I'd go with a squash-and-merge for a simpler commit history.

Over to @mdehoon to comment...

**Review by mdehoon — APPROVED** [review]

**Comment by pvalle6** [comment]
> One minor formatting change requested in the docstring example.
> 
> Unless you wanted to rebase this, I'd go with a squash-and-merge for a simpler commit history.
> 
> Over to @mdehoon to comment...

Completely agree, I'll update when I get to PC

**Review by peterjc — APPROVED** [review]
I fixed the indentation, good to squash-and-merge once tests finish running.

**Comment by peterjc** [comment]
Merged, thank you!

**Pull request merged**